src/app/board/quest/quest.component.ts

```diff
@@ -8,4 +9,36 @@ import { IQuest } from '../../shared/quest';
 })
 export class QuestComponent {
   @Input() quest: IQuest;
+
+  public isLoading: boolean = false;
+
+  constructor(
+    private questService: QuestService
+  ) {}
+
+  public acceptQuest(): void {
```

Comment:
The only difference between these methods is `true`/`false`. You can use one method instead of them